src/components/main/ForecastsDaily.tsx

```diff
@@ -50,7 +49,6 @@ const ForecastsDaily: FC<ForecastsDailyProps> = ({ view }) => {
         <option value="10">10 days</option>
         <option value="14">14 days</option>
       </select>
-      {loading && <Loader />}
```

Comment:
please remove `loading` from useAppSelector call result, no need to leave redundant variables